¤€<venv/lib/python3.6/site-packages/gevent/testing/exception.py 2pyXh